### PR TITLE
add integration tests against bundlers and browsers

### PR DESCRIPTION
**Pull request by bmeck**
Please CIRCLECI please work.

See https://github.com/DataDog/dd-trace-js/pull/1674

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -118,6 +118,11 @@ node-integration-base: &node-integration-base
   working_directory: ~/dd-trace-js
   steps:
     - checkout-and-yarn-install
+    - run:
+        name: Install Browser Dependencies
```
Comment (rochdev) [inline .circleci/config.yml:122]: Not all integration tests need those. Should this be an explicit browser test instead?

File: .circleci/config.yml [hunk 2]
```diff
@@ -118,6 +118,11 @@ node-integration-base: &node-integration-base
...
+        command: |
+          apt-get update -y
+          apt install -y fonts-liberation gconf-service libasound2 libatk1.0-0 libcairo2 libcups2 libfontconfig1 libgbm-dev libgdk-pixbuf2.0-0 libgtk-3-0 libicu-dev libjpeg-dev libnspr4 libnss3 libpango-1.0-0 libpangocairo-1.0-0 libpng-dev libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 xdg-utils
```
Comment (rochdev) [inline .circleci/config.yml:125]: Are these really all necessary?

File: LICENSE-3rdparty.csv [hunk 3]
```diff
@@ -23,6 +23,13 @@ require,retry,MIT,Copyright 2011 Tim Koschützki Felix Geisendörfer
 require,semver,ISC,Copyright Isaac Z. Schlueter and Contributors
 require,source-map,BSD-3-Clause,Copyright 2009-2011 Mozilla Foundation and contributors
 require,source-map-resolve,MIT,Copyright 2014-2020 Simon Lydell 2019 Jinxiang
+dev,@rollup/plugin-commonjs,MIT,Copyright 2019 RollupJS Plugin Contributors
```
Comment (rochdev) [inline LICENSE-3rdparty.csv:26]: These should be in alphabetical order to make the file easier to use. Also, bundlers are usually fairly heavy dependencies and can have a hard time with older versions of Node. Any way we can avoid having them in the main package.json?

**Review by rochdev — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (bmeck) [reply .circleci/config.yml:125]: sadly yes for running against a real browser

File: .circleci/config.yml (hunk 1 above)
Comment (bmeck) [reply .circleci/config.yml:122]: we could i guess but likely it would mean this isn't checked as often

File: .circleci/config.yml (hunk 2 above)
Comment (rochdev) [reply .circleci/config.yml:125]: Isn't there any one package that includes all or most of them? My experience with that many requirements to do only one thing is that it keeps breaking as the dependencies change over time.

File: .circleci/config.yml (hunk 1 above)
Comment (rochdev) [reply .circleci/config.yml:122]: Why would it be checked less often?

**Comment by tlhunter** [comment]
It looks like this PR ended up a bit stale and there were some open queries. Is this something you're still working on or is it safe to close for now?

**Comment by tlhunter** [comment]
I'll close this PR for now but we can consider reopening it in the future.

**Pull request closed without merging**